features/aggregate_build_logs.feature

```diff
@@ -1,7 +1,8 @@
 @wip
-Feature: Spotting and diagnosing software stack issues
+Feature: Aggregating Build Logs
     Scenario: Browsing logs of container images builds during Amun inspection jobs
-        Given Amun Service is deployed
+        Given amun service is accessible using HTTP
+        When an amun inspection job is submitted
```

Comment:
@fridex should I schedule only one inspection job or 2 here?

Reply:
one job, just as the example from the swagger UI